src/pages/references/changelog.md

```diff
@@ -27,6 +27,33 @@ contributors:
+  * If your code takes more than about 150-160ms to process, you may see warnings from the co-editing service in the console (`LECCoeditingService: Invalid sequenceNumber`). The cause for this is being researched and will be addressed in a future release. To reduce the number of times you see this error, reduce the length of time your script takes in a single editing frame — use `setTimeout` if necessary to batch your work.
+* Common pitfalls
+  * If you split your work on a document over multiple frames using `setTimeout`, be sure to protect against reentrancy, otherwise you may end up corrupting the user's undo stack. You should disable elements on the panel UI that could allow the user to execute your code before it is complete and then re-enable those elements when the code is done. The issue will be fixed in a future release. 
+  * Executing `await runtime.apiProxy("script")` from your script runtime code will do nothing, and nothing will be logged on the console.
+  * When adding shapes to a document, you should take care to ensure that the content is at least partially visible on the page. Otherwise the user may not have a way to edit it easily in the future.
+* Unexpected behavior
+  * If the user has a selection and your add-on creates new content, the selection is cleared. This will be addressed before release.
+  * When you add text content to a document, font substitution is not working correctly. This means that if you use Japanese characters, the user may see square boxes instead. If the user were to type the content manually, however, they would see the correct rendering. This issue will be fixed before release.
+  * Setting relative rotation on a group before adding any children is considered a "no-op" since these groups don't have a defined center point. Always modify group rotation after adding children.
+* Debugging
+  * If your script code has a syntax error, the console will log an unhelpful error message (similar to `Uncaught (in promise) at <adobe-internal.js:49>`). Your add-on panel UI will be visible and continue to be interactive, but it won't be able to communicate with the script runtime, resulting in what feels like non-responsive UI (e.g., clicking doesn't trigger the expected action). You'll want to configure your editor to highlight any syntax editors so that you can be sure your code is at least syntactically correct before you save.
+  * When you communicate to your panel from the script runtime, you may see `Empty transaction not added to pendingTransactions` in the browser log. A common use case might be to communicate progress if you're creating a lot of elements on the document. In this case, you should throttle how often you send progress updates to your panel so that you don't generate hundreds or thousands of these log messages (since this may get in the way of easily debugging any issues that might arise).
```

Comment:
We can remove this bullet if the use suggested wording above about ignoring certain console messages.

Reply:
Done